modules/social_features/social_profile/modules/social_profile_privacy/social_profile_privacy.install

```diff
@@ -63,3 +65,178 @@ function _social_profile_privacy_get_permissions($role) {
 
   return $permissions[$role];
 }
+
+/**
+ * Define visibility options of separate fields per user.
+ */
+function social_profile_privacy_update_8001(&$sandbox) {
```

Comment:
Do we also need these changes on `hook_isntall`?

Reply:
No, we don't. This update converts an old configuration for a new one but when a module will be enabled on a project then a new configuration is used.